watertap/unit_models/zero_order/anaerobic_digestion_reactive_zo.py

```diff
@@ -68,3 +68,29 @@ def biogas_prod(b, t):
                     to_units=pyunits.m**3 / pyunits.s,
                 )
             )
+
+        if self.config.process_subtype == "GLSD_anaerobic_digester":
```

Comment:
I think you can add a constraint here for biogas_production == sum(mass flow of byproduct stream gas components). Also, should only fix biogas_tss_ratio for the default subtype, but for this subtype, should leave it unfixed so that it is calculated by the biogas_prod constraint.

Reply:
Fair point-- we could resolve that by having some assumed value for density of the mixture, for example. Alternatively, perhaps we should only build the biogas_production and biogas_tss_ratio vars (and associated constraint) when the subtype is default (or when subtype is not "GLSD_anaerobic_digester")

Reply:
I've reassigned these biogas vars only to subtype == default or is None